python/lsst/ts/phosim/telescope/TeleFacade.py

```diff
@@ -324,14 +324,22 @@ def setInstName(self, camType, defocalDist=None):
         Raises
         ------
         ValueError
-            Defocal distance can not be <= 0.
+            Defocal distance can only be = 0 with LsstCam.
+        ValueError
+            Defocal distance can not be < 0.
         """
 
         if defocalDist is None:
             defocalDist = self.getDefaultDefocalDist()
 
-        if defocalDist <= 0:
-            raise ValueError("Defocal distance can not be <= 0.")
+        if (defocalDist == 0) & (camType != CamType.LsstCam):
```

Comment:
`if (defocalDist == 0) & (camType != CamType.LsstCam):` --> `if (defocalDist == 0) and (camType != CamType.LsstCam):`